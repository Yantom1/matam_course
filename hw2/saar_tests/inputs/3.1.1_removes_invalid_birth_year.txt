10000000, stude, 20, -2000, 1
20000000, stud, 19, 2001, 2
30000000, stu, 18, 2000, 3
40000000, st, 17, 2003, 2
50000000, s, 19, 2005, 4
