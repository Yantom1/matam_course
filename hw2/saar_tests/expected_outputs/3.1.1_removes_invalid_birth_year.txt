20000000, stud, 19, 2001, 2
40000000, st, 17, 2003, 2
